playbooks/roles/gitcontrol/files/github_api.py

```diff
@@ -18,6 +20,10 @@
 def read_yaml_file(path, org=None, endpoint=None, repo_name=None):
     if endpoint in ['manage_collaborators', 'branch_protection', 'options']:
         path += f'{org}/repositories/{repo_name}.yml'
+    if endpoint in ['teams']:
+        path += f'/{org}/teams/members.yml'
```

Comment:
please get rid of the starting "/" char. This explicitly hints that this must be a relative path